docs/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md

```diff
@@ -180,457 +180,1595 @@ RETURN example.func_example("first argument");
+
+### get\_vertex\_by\_id()
+
+```python
+def get_vertex_by_id(vertex_id: VertexId) -> Vertex
+```
+
+Return the Vertex corresponding to the given vertex_id from the graph.
+
+Access to a Vertex is only valid during a single execution of a
+procedure in a query. You should not globally store the returned
+Vertex.
+
+**Arguments**:
+
+- `vertex_id` - Memgraph Vertex ID
+  
+
+**Returns**:
+
+  `Vertex`corresponding to `vertex_id`
+  
+
+**Raises**:
+
+- `IndexError` - If unable to find the given vertex_id.
+- `InvalidContextError` - If context is invalid.
+  
+
+**Examples**:
+
+  ```graph.get_vertex_by_id(vertex_id)```
+
+### vertices
+
+```python
+@property
+def vertices() -> Vertices
+```
+
+Get all vertices in the graph.
+
+Access to a Vertex is only valid during a single execution of a
+procedure in a query. You should not globally store the returned Vertex
+instances.
+
+**Returns**:
+
+  `Vertices` that contained in the graph.
+  
+
+**Raises**:
+
+- `InvalidContextError` - If context is invalid.
+  
+
+**Examples**:
+
+  Iteration over all graph `Vertices`.
+  
+  ```
+  graph = context.graph
+  for vertex in graph.vertices:
+  ```
+
+### is\_mutable()
+
+```python
+def is_mutable() -> bool
+```
+
+Check if the graph is mutable. Thus it can be used to modify vertices and edges.
+
+**Returns**:
+
+  A `bool` value that depends if the graph is mutable or not.
+  
+
+**Examples**:
+
+  ```graph.is_mutable()```
+
+### create\_vertex()
+
+```python
+def create_vertex() -> Vertex
+```
+
+Create an empty vertex.
+
+**Returns**:
+
+  Created `Vertex`.
+  
+
+**Raises**:
+
+- `ImmutableObjectError` - If `graph` is immutable.
+- `UnableToAllocateError` - If unable to allocate a vertex.
+  
+
+**Examples**:
+
+  Creating an empty vertex.
+  ```vertex = graph.create_vertex()```
+
+### delete\_vertex()
+
+```python
+def delete_vertex(vertex: Vertex) -> None
+```
+
+Delete a vertex if there are no edges.
+
+**Arguments**:
+
+- `vertex` - `Vertex` to be deleted
+
+**Raises**:
+
+- `ImmutableObjectError` - If `graph` is immutable.
+- `LogicErrorError` - If `vertex` has edges.
+- `SerializationError` - If `vertex` has been modified by another transaction.
+
+**Examples**:
+
+  ```graph.delete_vertex(vertex)```
+
+### detach\_delete\_vertex()
+
+```python
+def detach_delete_vertex(vertex: Vertex) -> None
+```
+
+Delete a vertex and all of its edges.
+
+**Arguments**:
+
+- `vertex` - `Vertex` to be deleted with all of its edges
+  
+
+**Raises**:
+
+- `ImmutableObjectError` - If `graph` is immutable.
+- `SerializationError` - If `vertex` has been modified by another transaction.
+
+**Examples**:
+
+  ```graph.detach_delete_vertex(vertex)```
+
+### create\_edge()
+
+```python
+def create_edge(from_vertex: Vertex, to_vertex: Vertex,
+                edge_type: EdgeType) -> None
+```
+
+Create an edge.
+
+**Arguments**:
+
+- `from_vertex` - `Vertex` from where edge is directed.
+- `to_vertex` - `Vertex&#x27;  to where edge is directed.
```

Comment:
you accidentally wrote ' instead of \` so it isn't written in code style
![image](https://user-images.githubusercontent.com/95473291/177514773-ea45dff6-1892-40b0-894c-e01af3a12f2e.png)